### PR TITLE
Enhance README

### PR DESCRIPTION
**Pull request by dkj**
Give more context for non (or new) NPG team members.

### CODE REVIEW

**Review by marcomoscasgr — APPROVED** [review]
Looks good. The overview explains the main steps of the pipeline highlighting the per lane/tags workflow through deplexing.

File: README.md [hunk 1]
```diff
@@ -1,55 +1,137 @@
...
+Within this DAG there are two step which are key in producing the main data products:
 
-The graph represented by the input file should be a directed acyclic
-graph (DAG). Each graph node should have an id, which should be unique,
-and a label, which is the name of the pipeline step.
+* `p4_stage1_analysis` processes data at the lane level within a flowcell/run: includes conversion of instrument output (BCL files) to BAM format, demultiplexing of data within a lane to tagged libraries, alignment with any spiked phiX, (for some instrument types) detection of indel inducing fluidics bubbles and marking reads with fail bit, and (for some instrument types) detection and marking of sequencing adapter.
+* `seq_alignment` process data at tagged library, or lane and tagged library, level: includes alignment to the target genome (or not), a naive human read filtering capability, splitting of human target data by autosome/allosome capability, (for some instrument types) removal of marked adapter pre-alignment and pasting post-alignment for no loss of instrument basecalls or qualities, duplicate marking, and creation of standard sequencing metrics files.
```
[the remark this answers is not in the document]
Comment (dkj) [reply README.md:97]: if adapter basecall and sequence were permanent removed, then those basecalls and qualities would be permanently lost - but we're guaranteeing all basecalls and qualities from the instrument are provided to end customer so we have to put that information back in the BAM/CRAM records.

[the remark this answers is not in the document]
Comment (dkj) [reply README.md:97]: will amend slightly:
"and pasting post-alignment (so there is no loss of instrument basecalls or quality data),"

File: README.md [hunk 2]
```diff
@@ -1,55 +1,137 @@
...
-The files follow [JSON Graph Format](https://github.com/jsongraph/json-graph-specification)
-systax. Individual pipeline steps are defined as graph nodes, dependencies
-between them as directed graph edges. If step B should be executed
-after step A finishes, step B is is considered to be dependant on step A.
+Within this DAG there are two step which are key in producing the main data products:
 
-The graph represented by the input file should be a directed acyclic
-graph (DAG). Each graph node should have an id, which should be unique,
-and a label, which is the name of the pipeline step.
+* `p4_stage1_analysis` processes data at the lane level within a flowcell/run: includes conversion of instrument output (BCL files) to BAM format, demultiplexing of data within a lane to tagged libraries, alignment with any spiked phiX, (for some instrument types) detection of indel inducing fluidics bubbles and marking reads with fail bit, and (for some instrument types) detection and marking of sequencing adapter.
```
[the remark this answers is not in the document]
Comment (dkj) [reply README.md:96]: Yes: a tool now within `bambi` looks for high indel rate in phiX alignments for a particular area of the flowcell (x,y found by parsing the read names)

**Pull request merged**